aws-lambda/src/main.rs

```diff
@@ -39,11 +39,8 @@ async fn main() -> Result<(), Error> {
         .await?;
     let blueprint = Blueprint::try_from(&config)?;
 
-    let app_ctx = Arc::new(AppContext::new(
-        blueprint,
-        runtime,
-        config.extensions.endpoints,
-    ));
+    let app_ctx = AppContext::new(blueprint, runtime, config.extensions.endpoints).await?;
```

Comment:
Avoid performing side-effect while creating instances of structs. Let's add new method called `register_apollo_schema` which is an async method, that can be called on AppContext if required.

Reply:
removed `register` call since it's not working properly